projects/1.18/assets/lavatech/lavaplus/lang/zh_cn.json

```
@@ -0,0 +1,179 @@
+{
+  "item.lavaplus.lavabrick": "熔岩砖",
+  "message.lavabrick":"用于制造机器和狱火锭的温暖砖块。",
+
+  "item.lavaplus.infernalpickaxe": "狱火镐",
+  "message.infernalpickaxe":"一把坚固的镐，右键使用时会放置火把。",
+
+  "item.lavaplus.infernalsword": "狱火剑",
+  "message.infernalsword":"一把能点燃目标的剑。",
+
+  "item.lavaplus.infernalbrick": "狱火锭",
+  "message.infernalbrick":"非常热！",
+
+  "item.lavaplus.magmacoal": "岩浆煤炭",
+  "message.magmacoal":"燃烧时间是煤炭的两倍。在煤炭再加工机中制造。可用作焦煤的替代品。",

```

```suggestion
  "message.magmacoal":"燃烧时间是煤炭的两倍。可在煤炭再加工机中制造，并用于替代焦煤。",
```
全文都有这种[翻译腔](https://zh.wikipedia.org/wiki/Wikipedia:%E7%BF%BB%E8%AF%91%E8%85%94)的细节问题，建议自行修改，全文通查